src/test/java/org/folio/entitlement/support/base/BaseIntegrationTest.java

```diff
@@ -226,4 +228,8 @@ public static String getSystemAccessToken() {
     var accessToken = System.getProperty(SYSTEM_ACCESS_TOKEN_SYSTEM_PROPERTY_KEY);
     return accessToken != null ? accessToken : OKAPI_TOKEN;
   }
+
+  protected WireMock getWireMockClient() throws Exception {
```

Comment:
run wiremock in container using testcontainers, WireMockExtension can be helpful (from `folio-be-testing` library in `applications-poc-tools`)

Reply:
This is not a container, this is just a client that calls WireMock API